### PR TITLE
Format floats

### PR DESCRIPTION
**Pull request by lthiery**
Addresses #17

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -6,11 +6,10 @@
 
 {deps, [
     {clique, ".*", {git, "https://github.com/helium/clique.git", {branch, "develop"}}},
-    {lager, "3.8.1"},
```
Comment (michaeldjeffrey) [inline rebar.config:9]: no more lager?

Comment (Vagabond) [reply rebar.config:9]: It's pulled in from core

File: rebar.config [hunk 2]
```diff
@@ -6,11 +6,10 @@
...
     {observer_cli, "1.6.0"},
-    {jsx, "3.0.0"},
+    {jsx, {git, "https://github.com/lthiery/jsx", {branch, "main"}}},
     {kvc, {git, "https://github.com/etrepum/kvc", {tag, "v1.7.0"}}},
-    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "master"}}},
+    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {tag, "2021.09.16.1"}}},
```
Comment (Vagabond) [inline rebar.config:12]: Is this intentional?

Comment (lthiery) [reply rebar.config:12]: Hmm, yea I guess I could/should revert without further discussion. What's best practice here? It seems to be that explicit tagging would be favorable?

**Comment by lthiery** [comment]
Hold on this. It looks like the encode call is being made somewhere else since this has no effect in the app yet.

File: rebar.config (hunk 2 above)
Comment (lthiery) [reply rebar.config]: I reverted to branch "master"

**Comment by lthiery** [comment]
> Hold on this. It looks like the encode call is being made somewhere else since this has no effect in the app yet.

Fixed this and this is good to go.

**Review by michaeldjeffrey — APPROVED** [review]

**Pull request merged**